scripts/common_functions.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Function used to wait for node names according to given
```

Comment:
It seems to me that this function does not wait for anything, it just returns the node names based on some pattern.